cmd/collectors/rest/plugins/auditlog/auditlog.go

```diff
@@ -0,0 +1,252 @@
+package auditlog
+
+import (
+	"fmt"
+	"github.com/netapp/harvest/v2/cmd/collectors"
+	"github.com/netapp/harvest/v2/cmd/poller/plugin"
+	"github.com/netapp/harvest/v2/cmd/tools/rest"
+	"github.com/netapp/harvest/v2/pkg/conf"
+	"github.com/netapp/harvest/v2/pkg/matrix"
+	"github.com/netapp/harvest/v2/pkg/slogx"
+	"github.com/netapp/harvest/v2/pkg/util"
+	"log/slog"
+	"strings"
+	"time"
+)
+
+const (
+	auditMatrix             = "audit"
+	defaultDataPollDuration = 3 * time.Minute
+)
+
+type CacheRefresher interface {
+	RefreshCache() error
+}
+
+var defaultFields = []string{"application", "location", "user", "timestamp", "state"}
+var volumeCache = make(map[string]VolumeInfo)
+var volumeCacheCopy = make(map[string]VolumeInfo)
+var cacheForceRefresh bool
```

Comment:
Is force the right name since it seems like this variable prevents refreshes? Maybe a better name is `hasCacheRefreshed` or `hasRefreshedCache`? 

If I understand the life-cycle correctly, this variable will be reset each time `Run()` is called and then called once when the handler's `ExtractNames` runs.

If that's the case, would it make more sense to be more explicit and call RefreshCache() once inside the `if a.HasVolumeConfig()` block in the Run method?